src/hooks/useSortablePlaylist.ts

```diff
@@ -0,0 +1,45 @@
+import { useEffect, useRef } from 'react'
+import Sortable from 'sortablejs'
+import { getFirestore, doc, updateDoc } from 'firebase/firestore'
+
+export function useSortablePlaylist(
+  playlistData: any,
+  setPlaylistData: React.Dispatch<React.SetStateAction<any>>,
+  setCurrentVideoUrl: React.Dispatch<React.SetStateAction<string>>,
+  id: string | undefined,
+  isOwner: boolean
+) {
+  const db = getFirestore()
+  const itemRef = useRef<HTMLDivElement | null>(null)
+
+  useEffect(() => {
+    if (!itemRef.current || !playlistData || !playlistData.urls || !isOwner)
+      return
```

Comment:
P2;

1줄이 넘어가는 if문이라면 curly braces로 감싸 return합니다.
```suggestion
    if (!itemRef.current || !playlistData || !playlistData.urls || !isOwner) {
      return
    }
```

Reply:
https://jojoldu.tistory.com/6

위 글에 3번 참고해 주세요. <자바스크립트 핵심가이드> 같은 자바스크립트 바이블들(더글라스 크락포드 같은 유명인사들이 쓴 책)에서 공통적으로 금지하는 안티패턴입니다.